### PR TITLE
Added Layout test file

### PR DESCRIPTION
**Pull request by jhalak27**
For issue #403 

Added test file for fury/layout.py in fury/tests/test_layout.py

### CODE REVIEW

File: fury/tests/test_layout.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import numpy as np
+import numpy.testing as npt
+from fury import actor
+from fury.layout import GridLayout
+
+
+def test_grid_layout_gets_cell_shape():
+    centers = np.asarray([[0, 0, 0], [0, 0, 0]])
```
Comment (skoudoro) [inline fury/tests/test_layout.py:8]: Strange, Could you explain why both center are at `(0,0,0)`

File: fury/tests/test_layout.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    directions = np.asarray([[0, 0, 0], [0, 0, 0]])
```
Comment (skoudoro) [inline fury/tests/test_layout.py:9]: same comment as above

File: fury/tests/test_layout.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+    heights = np.asarray([2, 5])
+    colors = np.random.rand(2, 3)
+    cylinders = actor.cylinder(centers, directions, colors, heights=heights)
+    grid = GridLayout()
+    grid_square = GridLayout(cell_shape="square")
+    grid_diagonal = GridLayout(cell_shape="diagonal")
+    shape = grid.get_cells_shape([cylinders])
+    shape_square = grid_square.get_cells_shape([cylinders])
+    shape_diagonal = grid_diagonal.get_cells_shape([cylinders])
```
Comment (skoudoro) [inline fury/tests/test_layout.py:18]: can you add a screenshot of this grid in this PR? thank you

File: fury/tests/test_layout.py [hunk 4]
```diff
@@ -0,0 +1,33 @@
...
+    npt.assert_array_equal(shape, [[0.5, 5]])
+    npt.assert_array_equal(shape_square, [[5, 5]])
+    npt.assert_array_almost_equal(shape_diagonal, [[5, 5]], 0)
+
+
+def test_grid_layout_compute_positions():
+    centers = np.asarray([[0, 0, 0], [0, 0, 0]])
+    directions = np.asarray([[0, 0, 0], [0, 0, 0]])
```
Comment (skoudoro) [inline fury/tests/test_layout.py:26]: same comment as above

**Review by skoudoro — COMMENTED** [review]
Hi @jhalak27,

Thank you for doing this!

Could you also add tests for `Layout` class. I do not see any. After that, we will look deeper into your tests.  Please, could provide some screenshot that we get an idea of your tests grid. Thank you

Thank you

**Comment by skoudoro** [comment]
Hi @jhalak27, 

Any news concerning this PR? Please, let us know if you have any questions.

File: fury/tests/test_layout.py (hunk 1 above)
Comment (jhalak27) [reply fury/tests/test_layout.py:8]: There is no reason for selecting (0,0,0). I guess any center should do. Will change if needed.

Comment (jhalak27) [reply fury/tests/test_layout.py:8]: And, sorry for late response, had got a bit busy in coursework. :sweat_smile:

**Comment by jhalak27** [comment]
Hi @skoudoro,
I had added few replies to your comments earlier, can you have a look at them? I was waiting for your response to modify the test case.

**Comment by skoudoro** [comment]
superseded by #434. Closing. Thanks @jhalak27 for this effort!

**Pull request closed without merging**